src/lib.rs

```diff
@@ -83,12 +105,15 @@ pub fn run(args: Args) {
             .expect(&format!("Unknown capture name: `{}`", capture_name))
     });
     let output_mode = get_output_mode(&args);
+    let buffer_writer = BufferWriter::stdout(ColorChoice::Never);
 
-    get_project_file_walker(&*supported_language)
+    get_project_file_walker(&*supported_language, &args.use_paths())
         .into_parallel_iterator()
         .for_each(|project_file_dir_entry| {
-            let mut printer = get_printer(output_mode);
-            let path = format_relative_path(project_file_dir_entry.path(), output_mode);
+            let printer = get_printer(&buffer_writer, output_mode);
+            let mut printer = printer.borrow_mut();
```

Comment:
(See below comment for why this is in a `RefCell` in the first place) Specifically here my understanding of why this should never result in a runtime borrow error is that (like all `RefCell`'s) this `RefCell` is local to the current thread so as long as we drop it when we're done with it (and don't eg call back into this code path in a nested way) then there should be no way for a simultaneous borrow to occur